### DISCUSSION TITLE
Option to allow linebreaks in the default title field

### DISCUSSION THREAD

**Discussion by kevinmu17** [craftcms/cms#8939] [Ideas] [answered]
### Description
I like what you guys did with the new title field and how it is presenting itself now for the user! But i'm tumbling over the fact that there is no linebreak option here, so in, my situation, most cases i need to create another title field just to support this feature, so my projects contain 2 title fields which i'm not a big fan of. so, is it possible to give us the option to activate this?

![Screen Shot 2020-10-09 at 17 34 13](https://user-images.githubusercontent.com/8747373/95612094-1a0c8400-0a63-11eb-969d-d9e5c6f2276b.png)

**Comment by matthewstick** [comment]
Please add this feature

**Comment by spotd** [comment]
I assume this is not an easy fix? i regularly need this feature

**Reply by brandonkelly** [reply]
Can you give me an example of an entry title that would need a line break?

**Reply by matthewstick** [reply]
I'm being copied in this thread because I have a comment from 4 years ago ◡̈ 

We definitely do this all the time.  And still do.  It's super common and normal.

Basically title is in every single block and section - title is the default.  Often title is one line plain text. But frequently we want to use title more visually on the front end.  especially since title is in everything no matter what.  

For example - in a design element.  I want this ALWAYS on three lines. EXACTLY like this:
FOR YOUR
CONSIDERATON
2025

or this:

AN EXAMPLE
OF MANUALLY FORCING
LINE BREAKS IN 
TITLE
WHERE AUTOMATIC
WORD WRAP
WOULD NEVER
BE ABLE
TO MATCH 
MANUALLY ENTERED LINE BREAKS
BECAUSE CSS WORD WRAP
CAN'T HANDLE
USER DEFINED LINE BREAKS


Because title doesnt support line breaks, I've come to use title just for meta title on pages that have meta title.  Then in areas that need line breaks, I hide it and make a new field called label which is basically the same as title except it allows line breaks.

**Reply by kevinmu17** [reply]
Indeed like this ;)
Our partners delivering design elements that often have big font title elements that placed intentionally on ~ 3 lines
<img width="1414" height="529" alt="Screenshot 2025-08-26 at 09 55 37" src="https://github.com/user-attachments/assets/dec05b23-c213-45fc-8b91-c9710c1bfc29" />

**Comment by spotd** [comment]
Hi

I always get requests from the client to have this ability, mainly when the width of the text area won’t work universally. So say we are using a title that sits nicely over a design, and you also have multiple languages, having ability to force the line break so design stays consistent in all languages.

I have often had them put in ‘\n’ which isn’t great.

Cheers
Mark

Mark Dodson
Creative Director
mobile: + 44 7702573283
email: ***@***.***
web: www.spotd.uk

**Comment by spotd** [comment]
@brandonkelly any news on this one?

**Reply by brandonkelly** [reply]
We’ve discussed it internally and the general consensus is that we should make it possible.

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.9.0 is out with a new “Allow line breaks in titles” entry type setting (#18265) 🎉
